src/ai/backend/manager/services/processors.py

```diff
@@ -179,6 +187,9 @@ def create(cls, args: ProcessorArgs, action_monitors: list[ActionMonitor]) -> Se
         resource_preset_processors = ResourcePresetProcessors(
             services.resource_preset, action_monitors
         )
+        utilization_metric_processors = UtilizationMetricProcessors(
+            services.utilization_metric, action_monitors
+        )
```

Comment:
I wanted to tie the processor to all metrics, not just utilization. I'll pass it over.